tests/core/fixtures.py

```diff
@@ -29,3 +29,48 @@
     "88a012f977552ebdb7a492cf35f3106df16ccb4576ebad4113056ee1f52cbe49"
     "78c1c0"
 )
+
+# This is one header from each of the first 40 epochs, on Ropsten,
+# blocks: 1, 30001, 60001, etc
+ropsten_epoch_headers = [
+    '0xf90210a041941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2da01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d4934794d1aeb42885a43b72b518182ef893125814811048a0c7b01007a10da045eacb90385887dd0c38fcb5db7393006bdde24b93873c334ba056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000830f3a000183ffc001808458318da299d883010503846765746887676f312e372e318664617277696ea00f98b15f1a4901a7e9204f3c500a7bd527b3fb2c3340e12176a44b83e414a69e880ece08ea8c49dfd9',  # noqa: E501
```

Comment:
Plopping these in a flat text file with one per line is actually quite easy to load > decode.  That reduces these large blocks of hex in python files (at the cost of obscuring them in some `tests/core/ropsten_epoch_headers.rlp` file).  easy then to wrap them up in a fixture for easy use.

Don't feel very strongly on the subject, mostly just getting bothered here and there by large hex blobs in our python files but unsure if the situation can be improved.